### ISSUE TITLE
Support for graph 2.0

### ISSUE THREAD

**Issue by prabirshrestha** [facebook-csharp-sdk/facebook-csharp-sdk#294]
https://developers.facebook.com/docs/apps/upgrading/

```bash
curl https://graph.facebook.com/v2.0/me?access_token={valid-token}
```

>With the launch of v2.0 of the API, we have two versions in service at the same time:

>Version 1.0, which is what we call the API as it existed the day before v2.0 was launched. We'll support v1.0 for one year and it will expire on April 30th, 2015.
Version 2.0, which is what this upgrade guide covers. Version 2.0 is supported for at least two years. At the earliest, it will expire on April 30th, 2016.

**Comment by prabirshrestha** [comment]
Will leave it open for now until we finish up with all the unit tests.

**Comment by realworld666** [comment]
Whats the status of this? v1.0 will not be supported for much longer.

**Comment by marcstober** [comment]
@realworld666 It's not well documented but I got an answer to this on StackOverflow: http://stackoverflow.com/questions/28458334/how-to-call-versioned-graph-api-services-in-facebook-net-sdk?noredirect=1#comment45271071_28458334 

I'm able to call v2.2 of the Facebook API by setting the Version in my code now.

**Comment by prabirshrestha** [comment]
@realworld666 @marcstober Please refer to the PR for docs while it is in the preview. https://github.com/facebook-csharp-sdk/facebook-csharp-sdk/pull/311

**Comment by marcstober** [comment]
@prabirshrestha what's the "PR"?

**Comment by prabirshrestha** [comment]
@marcstober PR means pull request.

**Comment by WynsleyRiley** [comment]
i have software running against 'Facebook C# SDK'. 

I've tried to find out if I need to upgrade to a later version in ordere to be sure I wil not be dropped when the FB API V1 is dropped.

Also, I've tried the latest version and can't find a FBCliet.Version field when doing requests for Likes.

Can anyone help?

**Comment by prabirshrestha** [comment]
@WynsleyRiley You need to make sue you are downloading pre-release. 

https://www.nuget.org/packages/Facebook/7.0.5-beta

```powershell
Install-Package Facebook -Pre
```

**Comment by WynsleyRiley** [comment]
Thanks for your reply @prabirshrestha. Which version of .NET does the pre-release require, 4.5?

**Comment by WynsleyRiley** [comment]
Hi @prabirshrestha

I downloaded the pre-release using your command, Tested it in VS 13 Express and Professional with .Net V4.5.

The code is below, it fails when the following line is included, any ideas?

client.Version = "v2.2";

"Method not found: 'Void Facebook.FacebookClient.set_Version(System.String)'.

If a method in a class library has been removed or renamed, recompile any assemblies that reference that method.
This exception is typically thrown when an attempt is made to dynamically access a deleted or renamed method of an assembly that is not referenced by its strong name."

Thanks

Wyn
------------------------------------------

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Facebook;

namespace Test2
{
    public class Class1
    {
        public static void Main(string[] args)
        {
            Console.WriteLine("Started!");

            var client = new FacebookClient();
            //client.Version = "v2.2";

            Console.WriteLine("Finished!");
            Console.ReadKey();
        }
    }
}
